test/PoolManager.t.sol

```diff
@@ -366,25 +366,25 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
     }
 
     function test_addLiquidity_gas() public {
-        snapStart("addLiquidity");
+        snapStart("PoolManager.addLiquidity");
         modifyLiquidityRouter.modifyLiquidity(key, LIQ_PARAMS, ZERO_BYTES);
         snapEnd();
     }
 
     function test_removeLiquidity_gas() public {
-        snapStart("removeLiquidity");
+        snapStart("PoolManager.removeLiquidity");
         modifyLiquidityRouter.modifyLiquidity(key, REMOVE_LIQ_PARAMS, ZERO_BYTES);
         snapEnd();
     }
 
     function test_addLiquidity_withNative_gas() public {
-        snapStart("addLiquidity with native token");
+        snapStart("PoolManager.addLiquidityWithNativeToken");
         modifyLiquidityRouter.modifyLiquidity{value: 1 ether}(nativeKey, LIQ_PARAMS, ZERO_BYTES);
         snapEnd();
     }
 
     function test_removeLiquidity_withNative_gas() public {
-        snapStart("removeLiquidity with native token");
+        snapStart("PoolManager.removeLiquidityWithNativeToken");
```

Comment:
I think it would be better to go
`ContractName.functionName_extraNotes`
e.g.
`PoolManager.removeLiquidity_withNative`
`PoolManager.swap_simple`
`PoolManager. collectProtocolFees_withNative`